packages/fast-components-react-msft/src/action-trigger/examples.data.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { ActionTrigger, ActionTriggerAppearance, ActionTriggerProps } from "./index";
+import schema from "./action-trigger.schema.json";
+import Documentation from "./.tmp/documentation";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import ReactSVGElementSchema from "../../app/components/react-svg-element.schema.json";
+
+const testDestination: string = "https://www.microsoft.com/en-us/";
+
+const glyphExample: any = {
+    id: ReactSVGElementSchema.id,
+    props: {},
+};
+
+export default {
+    name: "Action trigger",
+    component: ActionTrigger,
+    schema: schema as any,
+    documentation: <Documentation />,
+    detailData: {
+        children: "Action Trigger",
+        appearance: ActionTriggerAppearance.primary,
+        href: testDestination,
+        glyph: glyphExample,
+    },
+    data: [
+        {
+            appearance: ActionTriggerAppearance.primary,
+            children: "Primary action trigger",
+            href: testDestination,
+            glyph: glyphExample,
+            "data-sketch-symbol": "Action trigger - primary",
+        } as any, // TODO https://github.com/Microsoft/TypeScript/issues/6579
```

Comment:
This reads like the TODO is on the object, but that doesn't make sense to me as the linked issue is about a regex validated string issue. Can you elaborate on the TODO for me?

Reply:
I copied it over from call to action's example data file figuring that if it applied there it would also apply to the same code in action trigger.

Reply:
Interesting - I'll see if I can track down who added this. Casting the same thing in call to action as `CallToActionProps` also eliminated the type error. All that aside, we should ensure the issue is valid for this scenario as well (but I totally get the logic)

Reply:
Since nothing is specifically broken I think we should remove it.